src/main.cpp

```diff
@@ -189,6 +189,13 @@ DRAM_ATTR bool g_bUpdateStarted = false;
 DRAM_ATTR AppTime g_AppTime;                                                        // Keeps track of frame times
 DRAM_ATTR bool NTPTimeClient::_bClockSet = false;                                   // Has our clock been set by SNTP?
 
+#if USE_MATRIX
+    extern int g_MatrixPowerMilliwatts;                                             // Matrix power draw in mw
+    extern uint8_t g_MatrixScaledBrightness;                                        // 0-255 scaled brightness to stay in limit
+#endif
+
+extern DRAM_ATTR std::unique_ptr<EffectManager<GFXBase>> g_ptrEffectManager;       // The one and only global effect manager

```

Comment:
I don't think we need this any more. It's now part of SystemContainer (g_ptrSystem).

```suggestion
```